### PR TITLE
Change IECC plots to Census Region

### PR DESCRIPTION
**Pull request by joseph-robertson**
## Pull Request Description

Add `Census Region.tsv` to `project_testing` so that we can replace the "IECC" comparison plot with a "census region" comparison plot. Change `project_testing` locations to LA, Cook County, Harris County, NY.

Checklist:
- [x] Add project_testing's `Census Region.tsv` to resstock-estimation https://github.com/NREL/resstock-estimation/pull/179

## Checklist

Not all may apply:

- [ ] Unit tests have been added or updated
- [ ] All rake tasks have been run, and pass
- [ ] Documentation has been modified appropriately
- [ ] Any new options are added to `project_testing`
- [ ] `project_testing` runs without any failures
- [ ] No unexpected regression test changes
- [ ] All tests are passing
- [ ] The [changelog](https://github.com/NREL/resstock/blob/develop/CHANGELOG.md) has been updated appropriately
- [ ] This branch is up-to-date with develop

For more information on how to perform these checklist items, see the documentation's [Advanced Tutorial](https://resstock.readthedocs.io/en/latest/advanced_tutorial/index.html).

### CODE REVIEW

File: project_testing/housing_characteristics/Census Region.tsv [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Dependency=County	Option=Midwest	Option=Northeast	Option=South	Option=West	sampling_probability
+AZ, Maricopa County	0.000000	0.000000	0.000000	1.000000	0.166667
+CA, Los Angeles County	0.000000	0.000000	0.000000	1.000000	0.166667
+GA, Fulton County	0.000000	0.000000	1.000000	0.000000	0.166667
+IL, Cook County	1.000000	0.000000	0.000000	0.000000	0.166667
+TX, Harris County	0.000000	0.000000	1.000000	0.000000	0.166667
+WA, King County	0.000000	0.000000	0.000000	1.000000	0.166667
```
Comment (shorowit) [inline project_testing/housing_characteristics/Census Region.tsv:7]: I don't remember how we ended up with these 6 locations but can we revise them? Right now we have 1 midwest, 0 northeast, 2 south, and 3 west. What if we switch to 4 (or a multiple of 4) locations, such that these regions are equally represented?

Comment (joseph-robertson) [reply project_testing/housing_characteristics/Census Region.tsv:7]: @shorowit How about:
County | Census Region
-- | --
AZ, Maricopa County | West
IL, Cook County | Midwest
TX, Harris County | South
NY, New York County | Northeast

Comment (shorowit) [reply project_testing/housing_characteristics/Census Region.tsv:7]: Can you post the IECC climate zones for them? I'm guessing that NY & Cook County are similar climate zones, and same for Maricopa and Harris. It'd be nice to get more diversity of climate zones.

Comment (joseph-robertson) [reply project_testing/housing_characteristics/Census Region.tsv:7]: I updated the comment above. Perhaps we'd want one county per IECC climate zone (1A - 7B; 15 locations)?

Comment (shorowit) [reply project_testing/housing_characteristics/Census Region.tsv:7]: I don't have a strong opinion on this, but I think our requirements are that we have a diversity of climate zones and an equal number of locations in each Census Region. If 4 locations, that would be Harris County, LA, NY, and Cook County. But if you wanted to add more, I wouldn't object.

**Review by shorowit — APPROVED** [review]

**Pull request merged**